implementations/rust/ockam/ockam_command/src/node/create.rs

```diff
@@ -41,10 +47,20 @@ pub struct CreateCommand {
 impl CreateCommand {
     pub fn run(opts: CommandGlobalOpts, command: CreateCommand) {
         let cfg = &opts.config;
-        let address: SocketAddr = command
-            .api_address
-            .parse()
-            .expect("failed to parse api address");
+        let address: SocketAddr = if &command.api_address == "127.0.0.1:0" {
```

Comment:
Should check for the port value here? Ip address can be different (or replaced with domain name)
@mrinalwadhwa @tmuro17